specifyweb/businessrules/tests/geography.py

```diff
@@ -39,6 +40,31 @@ def test_delete_blocked_by_agentgeography(self):
         models.Agentgeography.objects.filter(geography=geography).delete()
         geography.delete()
 
+    def test_isaccepted_on_save(self):
+        earth = models.Geography.objects.create(
+            name="Earth",
+            definition=self.geographytreedef,
+            definitionitem=self.geographytreedef.treedefitems.all()[0])
+
+        continent = earth.definitionitem.children.create(
+            name="Continent",
+            treedef=earth.definition,
+            rankid=earth.definitionitem.rankid+100)
```

Comment:
Hmm is this needed, given that tree rank code should automatically set this? Or did you want to test this in isolation?

Reply:
Yep, not needed. 
I would advocate for more thorough, isolated tests for the changes in #4257.